dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/DataSourceService.java

```diff
@@ -659,9 +659,7 @@ protected void init() throws Exception {
                 }
             }
 
-            //TODO remove noship/beta guard before GA
-            isUCP = vendorImplClassName.startsWith("oracle.ucp.jdbc.") && 
-                            (wProps.containsKey("internalDevNonshipFunctionDoNotUseProduction") || (vProps.getFactoryPID() != null && vProps.getFactoryPID().equals("com.ibm.ws.jdbc.dataSource.properties.oracle.ucp")));
+            isUCP = vendorImplClassName.startsWith("oracle.ucp");
```

Comment:
do we want to continue gating this on `<properties.oracle.ucp>`?

Reply:
No, the check was only there to ensure this function could only be used when the beta properties.oracle.ucp element was used.  We now want this behavior whenever UCP is used.